### PR TITLE
Update QQ.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

File: src/chrome/content/rules/QQ.xml [hunk 1]
```diff
@@ -273,6 +274,7 @@
 	<target host="mail.qq.com" />
 	<target host="en.mail.qq.com" />
 	<target host="i.mail.qq.com" />
+	<target host="pingjs.qq.com />
```
Comment (fuglede) [inline src/chrome/content/rules/QQ.xml]: Missing a <code>"</code> here.

**Comment by J0WI** [comment]
I get a timeout on https://dp3.qq.com/.

**Comment by renyouguo** [comment]
@J0WI I'm ok.
Chinese site is too slow to get for foreigners sometime.

**Comment by J0WI** [comment]
Now I'm getting a mismatch on that host.

**Comment by J0WI** [comment]
Please document why you made these exclusions.

**Comment by J0WI** [comment]
@renyouguo do you still have timeouts in tests after #4810 has been fixed?

**Comment by renyouguo** [comment]
@J0WI https://travis-ci.org/EFForg/https-everywhere/jobs/133578823

**Comment by J0WI** [comment]
Okay, I think this is not related with #4810, but the error on https://travis-ci.org/EFForg/https-everywhere/jobs/133578823#L362 looks very strange.
cc @fuglede

**Comment by fuglede** [comment]
Very strange indeed; https://github.com/curl/curl/issues/619 perhaps?

**Comment by renyouguo** [comment]
@J0WI @fuglede #4919 Time out error.

**Comment by renyouguo** [comment]
https://travis-ci.org/EFForg/https-everywhere/jobs/133604786#L429
Time out again.

**Comment by renyouguo** [comment]
https://travis-ci.org/EFForg/https-everywhere/jobs/133606626#L354
No output.

**Comment by renyouguo** [comment]
@J0WI @fuglede Would you please merge this PR and #4963 firstly? In my 35 open PRs, only they fix the broken functions.
They are so important for chinese.

**Comment by J0WI** [comment]
I would love to merge this, but there are still some non documented removal of existing hosts in this PR.

**Comment by renyouguo** [comment]
@J0WI Which hosts?

**Comment by J0WI** [comment]
`boss.exmail.qq.com` and `support.exmail.qq.com`.
Note that some comments still refer to the removed `support.exmail.qq.com`.

**Comment by renyouguo** [comment]
@J0WI They do not exist.
Test:
http://ping.eu/ping/
http://tool.chinaz.com/speedtest.aspx
boss
support

File: src/chrome/content/rules/QQ.xml [hunk 2]
```diff
@@ -336,7 +329,6 @@
 	<!--securecookie host="^\.aq\.qq\.com$" name="^aq_base_sid$" /-->
 	<!--securecookie host="^\.exmail\.qq\.com$" name="^(?:edition|mcookie|ssl_edition)$" /-->
 	<!--securecookie host="^\.support\.exmail\.qq\.com$" name="^ssl_edition$" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/QQ.xml]: another reference to obsolete host

**Comment by J0WI** [comment]
They timeout here on https but also on http, so I'm okay with removing them.
But there is still an obsolete comment left with a reference to one of them.

**Comment by renyouguo** [comment]
@J0WI Maybe you need Python3 : https://travis-ci.org/EFForg/https-everywhere/jobs/134628375#L324
:)

**Comment by J0WI** [comment]
This finally passed all tests, changes lgtm.

**Pull request merged**